man/fitNullModel.Rd

```diff
@@ -112,9 +111,6 @@ isNullModelSmall(null.model)
     \item{zeroFLAG}{A vector of logicals the same length as \code{varComp} specifying whether the corresponding variance component estimate was set to 0 by the function due to convergence to the boundary in the AIREML procedure.}
     \item{RSS}{The residual sum of squares from the model fit. When \code{family} is gaussian, this will typically be 1 since the residual variance component is estimated separately.}
     \item{RSS0}{the sum of the `Ytilde` values squared. This is the sum of squared residuals under the null hypothesis of no genetic effect for the covariate and random effect adjusted model using the Frisch-Waugh-Lovell theorem.}
-    \item{model}{The model string.}
-    \item{outcome}{The outcome variable name.}
-    \item{covars}{A vector of covariate names}
 }
 
 The \code{fit} data frame contains the following columns, depending on which type of model was fit:
```

Comment:
@amstilp 
In line 120, it looks like the description of X*beta got lost. 
The description "where Z is the design matrix and beta are the effect size estimates for the fixed effects. Z*u represents the effects captured by the random effects specified with the cov.mat input." should be moved down to `resid.conditional` where we use those terms

Reply:
Thanks, fixed.